types/dsinjs__binary-tree/tslint.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "max-line-length": {
+            "options": [180]
+        },
+        "new-parens": true,
+        "no-conditional-assignment": true,
+        "no-consecutive-blank-lines": false,
```

Comment:
Why suppress lint rules?

Reply:
I will use the default constraint values for these  3 rules